classes/hook_callbacks.php

```diff
@@ -53,4 +53,19 @@ public static function before_footer_html_generation(\core\hook\output\before_fo
 
         tool_abconfig_execute_js('footer');
     }
+
+    /**
+     * Runs after config has been set.
+     *
+     * @param \core\hook\before_http_headers $hook
```

Comment:
Can you please fix this @param type, I think ci complain about this

Reply:
Thanks for catching that, updated this one.